coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "covered_percent": 100.0
+  }
+}
```

Comment:
Can you delete these files (git rm -- cached, josh will run you through it ) and then repush